Spec/RealtimeClientChannel.swift

```diff
@@ -70,11 +67,11 @@ class RealtimeClientChannel: QuickSpec {
                     }
```

Comment:
I think line 49 above (can't comment on it) also needs a `toEventually`, or a wait until for the `channel2.attach()` callback.